packages/lake/src/components/LakeCombobox.tsx

```diff
@@ -266,7 +270,7 @@ const LakeComboboxWithRef = <I,>(
         onDismiss={close}
         referenceRef={ref}
         autoFocus={false}
-        returnFocus={true}
+        returnFocus={false}
```

Comment:
Returning focus to the input on popover closed caused the never closing popover issue, as the popover appears again when focus is gain.
As there's no mode where the input is focused but the popover not visible, we can't use this prop in this particular context.